Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -682,4 +689,171 @@ ScriptOrModule VM::get_active_script_or_module() const
     return m_execution_context_stack[0]->script_or_module;
 }
 
+VM::StoredModule* VM::get_stored_module(ScriptOrModule const&, String const& filepath)
+{
+    auto end_or_module = m_loaded_modules.find_if([&](StoredModule const& stored_module) {
+        if (stored_module.filepath != filepath)
+            return false;
+        return true;
+        //        return stored_module.referencing_script_or_module.visit(
+        //            [&](Empty) {
+        //                return script_or_module.has<Empty>();
+        //            },
+        //            [&](Script* script) {
+        //                return script_or_module.has<Script*>() && script_or_module.get<Script*>() == script;
+        //            },
+        //            [&](Module* module) {
+        //                return script_or_module.has<Module*>() && script_or_module.get<Module*>() == module;
+        //            });
```

Comment:
Commented out code - left from debugging?

Reply:
This is a bit tricky because the spec says:
`Each time this operation is called with a specific referencingScriptOrModule, specifier pair as arguments it must return the same Module Record instance if it completes normally.`

So perhaps we need to search with both at some point. But for now this works so I have added a comment.